modules/rosa-configure-privatelink-configuration.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * cloud_infrastructure_access/rosa-aws-privatelink.adoc
+
+[id="rosa-configure-privatelink-configuration.adoc_{context}"]
+= Creating a PrivateLink cluster
+
+There are two types of AWS PrivateLink clusters, single Availability Zone (single-AZ) and Multi Availability Zone (Multi-AZ). This procedure describes creating a new cluster for each type using the *rosa* CLI.
+
+[NOTE]
+====
+You can enable PrivateLink cluster management only when creating a new Red Hat OpenShift Service on an AWS cluster.
+====
+
+.Prerequisites
+* Configure the *rosa* CLI and the AWS CLI environments. 
+
+== Creating a Single-AZ PrivateLink cluster
+.Procedure
+
+* To create a new Single-AZ cluster, enter the following command:
+[source,terminal]
+----
+$ rosa create cluster --private-link --cluster-name <cluster-name> --machine-cidr=<VPC CIDR>/16 --subnet-ids=<private-subnet-id>
+----
+
+== Creating a Multi-AZ PrivateLink cluster
+.Procedure
+* To create a new Multi-AZ cluster, enter the following command:
```

Comment:
s/To create a new Multi-AZ cluster, enter the following command:/Create a new Multi-AZ cluster: